editor/src/components/canvas/canvas-strategies/canvas-strategies.tsx

```diff
@@ -233,58 +235,69 @@ export function applyCanvasStrategy(
   return strategy.apply(canvasState, interactionSession, strategyState)
 }
 
-export const useDelayedCurrentStrategy = () => {
+export function useDelayedStrategyWithCallback<T>(
+  innerCallback: StateSelector<EditorStorePatched, T | null>,
```

Comment:
Please can you rename this, as it is a selector and not a callback (so this could lead to some amount of confusion)

Reply:
fixed